pkg/apiserver/authorization/authorizerfactory/builtin.go

```diff
@@ -1,27 +1,27 @@
 /*
- *
- * Copyright 2020 The KubeSphere Authors.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- * /
- */
+Copyright 2016 The Kubernetes Authors.
```

Comment:
So this file is from K8s?

Reply:
Following code copied from k8s.io/apiserver/pkg/authorization/authorizerfactory to avoid import collision.